Sources/SpeziOnboarding/Consent/Platform.swift

```diff
@@ -0,0 +1,23 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+#if canImport(UIKit)
+import class UIKit.UIFont
+import class UIKit.UIColor
+/// :nodoc:
```

Comment:
Does make sense to add a `@_documentation(visibility:)` annotation here and for the other types?

Reply:
yes (i originally decided not to do this because i wanted these to be included in the documentation, but i guess the names are self-documenting enough...)